cylc/flow/network/schema.py

```diff
@@ -1712,12 +1719,17 @@ class Arguments:
 class Stop(Mutation):
     class Meta:
         description = sstrip('''
-            Tell a scheduler to shut down,
-            or stop a specified flow from spawning any further.
```

Comment:
Are you sure this detail should be lost?

Reply:
Not in the least. Will put it back.